### ISSUE TITLE
Stuck on "Devon is gathering himself..." in GUI

### ISSUE THREAD

**Issue by krishmoran** [entropy-research/Devon#78]
Devon will produce a few outputs and then gets stuck at "Devon is gathering himself", ive left it sitting for an hour and still nothing. I'm using GPT-4o, don't seem to be getting rate limited, and in the terminal im getting:

Server: INFO:     127.0.0.1:51512 - "GET /sessions/UI/state HTTP/1.1" 200 OK

On restart it does the same thing, few outputs then stuck.

Any ideas here?

**Comment by ObjectJosh** [comment]
Hey @krishmoran yeah if it's stuck like that for more than 5 seconds the backend probably failed, so no need to wait any longer (we're pushing out a patch to properly display the error). For now, try opening the devtools and check the server error log. Sorry about that!

**Comment by akiradev0x** [comment]
@krishmoran if you ran it using the npx command from a terminal you should also be able to see the issue there. if you have a stack trace that could help!

**Comment by lsc64** [comment]
Sometimes it's stuck on a command. I've tried developing a web application and the command to start the webserver would (obviously) never return. So I manually had to kill it to continue iterating.

**Comment by ObjectJosh** [comment]
@lsc64 yeah, we noticed it too. In the next update it should be able to create background jobs! Thanks for bringing this up
